### DISCUSSION TITLE
What are the files in /data?

### DISCUSSION THREAD

**Discussion by wsw70** [dani-garcia/vaultwarden#3794] [Q&A] [answered]
I have a rather unusual question: **what are the Vaulwarden files in `/data`?**

The reason for my question is that it is late at night and I wildly imagined that it would be the best moment to set up a Minecraft server. The only problem is that I wrote `vaultwarden` when thinking `minecraft` (they are close, right?) and bam! I have my Vaultwarden `/data` full of square world artifacts...

I could recover from backup but if the list of the files is not large I will be better off just deleting the extra ones.

**Comment by jjlin** [comment, marked as answer]
See https://github.com/dani-garcia/vaultwarden/wiki/Backing-up-your-vault
